modelservice/src/main/java/com/expedia/adaptivealerting/modelservice/providers/graphite/GraphiteRequest.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2018-2019 Expedia Group, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.expedia.adaptivealerting.modelservice.providers.graphite;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class GraphiteRequest {
+    private String target;
+
+    public Map<String, Object> toParams() {
+        Map<String, Object> params = new HashMap<>();
+        params.put("target", target);
+        return params;
+    }
```

Comment:
Not necessary to generate a param map whose JSON is identical to the `GraphiteRequest` itself. Just use the `GraphiteRequest`. Could just pass a singleton map as well.